src/volcanic_island_simulator.py

```diff
@@ -119,6 +121,11 @@ def __init__(self, params={}):
         self.sea_level_mean = sea_level_params["mean"]
         self.sea_level_amplitude = sea_level_params["amplitude"]
         self.sea_level_period = sea_level_params["period"]
+
+        if "tectonics" in params:
+            self.uplift = params["tectonics"]["uplift"]
```

Comment:
I'm not sure if we need ["uplift"] here; my understanding is that if "tectonics" is a dictionary in params, and that dictionary only has a single entry ("uplift"), then simply writing:

self.uplift = params["tectonics"] 

will automatically unpack the info we want to access.

I could be wrong, though, so might want to get a second review on that.